File: app/src/main/java/com/chess/cryptobot/model/response/ErrorResponse.java
package com.chess.cryptobot.model.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ErrorResponse {
    @SerializedName("errorMessage")
    @Expose
    private String errorMessage;

    @SerializedName("message")
    @Expose
    private String message;

    public String getErrorMessage() {
        if (errorMessage == null) return message;
        return errorMessage;
    }
}

File: app/src/main/java/com/chess/cryptobot/model/response/ErrorResponse.kt
package com.chess.cryptobot.model.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ErrorResponse {
    @SerializedName("errorMessage")
    @Expose
    val errorMessage: String = "no error Message"
        get() = message?:msg?:code?:field
    @SerializedName("message")
    @Expose
    private val message: String? = null
    @SerializedName("msg")
    @Expose
    private val msg: String? = null
    @SerializedName("code")
    @Expose
    private val code: String? = null

}